src/main/java/net/syscon/elite/api/resource/OffenderResource.java

```diff
@@ -71,11 +71,11 @@
     @GetMapping("/{offenderNo}/addresses")
     @ApiOperation(value = "Return a list of addresses for a given offender, most recent first.")
     @ApiResponses(value = {
-            @ApiResponse(code = 200, message = "OK", response = OffenderAddress.class, responseContainer = "List"),
+            @ApiResponse(code = 200, message = "OK", response = AddressDto.class, responseContainer = "List"),
```

Comment:
I might be wrong on this one but I don't think we need to add the 200 response docs anymore. @petergphillips ???

Reply:
You could take it off and just start up the app and go the docs on localhost...

Reply:
Good catch, missed this one in the rename!